From f953c6648a7b449b2089ff0708fa880554d60d2e Mon Sep 17 00:00:00 2001
From: Benda Xu <heroxbd@gentoo.org>
Date: Wed, 10 Sep 2014 10:00:43 +0900
Subject: disable known failed tests

Bug: http://code.google.com/p/casacore/issues/detail?id=67
Description: Testsuite of tEarthField, tEarthMagneticMachine,
    tMEarthMagnetic are out of date with casacore-data. Data needed
    for tAntennaResponses is not available.
---
 images/Images/test/CMakeLists.txt     | 1 -
 measures/Measures/test/CMakeLists.txt | 3 ---
 2 files changed, 4 deletions(-)

diff --git a/images/Images/test/CMakeLists.txt b/images/Images/test/CMakeLists.txt
index 6ab01a7..8afff03 100644
--- a/images/Images/test/CMakeLists.txt
+++ b/images/Images/test/CMakeLists.txt
@@ -78,7 +78,6 @@ dImageMoments
 dImageStatistics
 dImageSummary
 dPagedImage
-tAntennaResponses
 tComponentImager
 tExtendImage
 tFITSExtImage
diff --git a/measures/Measures/test/CMakeLists.txt b/measures/Measures/test/CMakeLists.txt
index f7eb207..7e9c6fb 100644
--- a/measures/Measures/test/CMakeLists.txt
+++ b/measures/Measures/test/CMakeLists.txt
@@ -1,10 +1,7 @@
 set (tests
 dM1950_2000
 dMeasure
-tEarthField
-tEarthMagneticMachine
 tMBaseline
-tMEarthMagnetic
 tMeasComet
 tMeasJPL
 tMeasMath
